src/main/resources/static/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Getting Started: Serving Web Content</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+</head>
+<body>
+    <p>Waaaaaaazap</p>
+</body>
+</html>
```

Comment:
This is our home page ... spring automatically finds things in this directory apparently. Next thing to try out: https://spring.io/guides/gs/spring-boot/